templates/openshift/rbac/role.yaml

```diff
@@ -5,4 +5,4 @@ metadata:
 rules:
   - apiGroups: ['*']
     resources: ['*']
-    verbs: ['get', 'list']
+    verbs: ['get', 'list', 'create']
```

Comment:
through trial and error I found out that this verb is required for `oc cp`

Reply:
I have added a specific permission [here](https://github.com/integr8ly/backup-container-image/pull/6/files#diff-1857ec47aa300369db1c2fa542f40842R9) too

Reply:
@sedroche yours permissions were better, so I stolen them :)